.github/workflows/build_multiarch_images.yaml.yaml

```diff
@@ -10,6 +10,7 @@ jobs:
     name: Build Rocks and Push Arch Specific Images
     uses: canonical/k8s-workflows/.github/workflows/build_rocks.yaml@main
     with:
+      runs-on: "[self-hosted, Linux, ARM64, jammy, large]"
```

Comment:
try putting these labels on the `platform-labels` for `arm64`

Reply:
something like:

```
      platform-labels: '{"arm64": ["self-hosted", "Linux", "ARM64", "jammy", "large"]}'
```

Reply:
Done, thanks!